python/sgp4_scanner_stats.py

```diff
@@ -0,0 +1,165 @@
+                      'velocityErrorZ',                                                           \
+                      'velocityErrorMagnitude' ]
+
+print "Fetch successful!"
+print ""
+
+magnitudeTypes = [ 'positionErrorMagnitude', 'velocityErrorMagnitude' ]
+errorTypes = [ 'position error', 'velocity error' ]
+for loopIndex in range( 2 ):
+  errorType = errorTypes[ loopIndex ]
+  print ""
+  print "Stats for " + errorType + ":"
+  magnitudeError = scan_data[ magnitudeTypes[ loopIndex ] ]
+
+  # Calculate mean, variance and standard deviation
+  mu = sum( magnitudeError ) / len( magnitudeError )
+  print 'Mean = ' + repr( mu )
+
+  sumOfSquareDeviations = sum( ( x - mu )**2 for x in magnitudeError )
+  variance = sumOfSquareDeviations / len( magnitudeError )
+  print 'Variance = ' + repr( variance )
+
+  sigma = variance**0.5
+  print 'Standard Deviation = ' + repr( sigma )
+
+  # Calculate median
+  median = np.median( magnitudeError )
+  print 'median = ' + repr( median )
+
+  # Get maximum and minimum error values
+  maxError = max( magnitudeError )
+  print 'max error = ' + repr( maxError )
+  minError = min( magnitudeError )
+  print 'min error = ' + repr( minError )
+
+  # Get interquartile range
+  q75, q25 = np.percentile( magnitudeError, [ 75, 25 ] )
+  interquartileRange = q75 - q25
+  print 'IQR = ' + repr( interquartileRange )
+
+  # Print data to a csv file
+  orbitType = config['orbit_type']
+  file_path = path.relpath( config['output'] )
+  with open( file_path, 'ab' ) as csvfile:
+    fieldnames = [ 'Orbit Type', 'Error Type', 'Mean', 'Variance', 'Standard Deviation',          \
+                   'median', 'Max. Error', 'Min. Error', 'Interquartile Range' ]
+    dataWriter = csv.DictWriter( csvfile, fieldnames=fieldnames, extrasaction='raise' )
+
+    dataWriter.writeheader()
+    dataWriter.writerow( { 'Orbit Type'           : orbitType,                                    \
+                           'Error Type'           : errorType,                                    \
+                           'Mean'                 : mu,                                           \
+                           'Variance'             : variance,                                     \
+                           'Standard Deviation'   : sigma,                                        \
+                           'median'               : median,                                       \
+                           'Max. Error'           : maxError,                                     \
+                           'Min. Error'           : minError,                                     \
+                           'Interquartile Range'  : interquartileRange } )
+
+# Close SQLite database if it's still open.
+if database:
+    database.close( )
+
+# Stop timer
+end_time = time.time( )
+
+# Print elapsed time
+print "Script time: " + str("{:,g}".format(end_time - start_time)) + "s"
```

Comment:
Maybe add some dashed line before printing the Script time: because now it is not clear where the data you output to the terminal ends.